First version of the README.md file

KamVya2926 opened the pull request:
Hey Team,

I've had a go at the READMe file. Could you please take a look at it and let me know if you have any suggestions for improvements.

KamVya2926 committed:
First version of the README.md file

ayana-manoj commented on README.md:
```diff
@@ -1,2 +1,70 @@
+<img src="https://github.com/ayana-manoj/ClinVar_Search/blob/develop/docs/Overview.png" >
+
+Users can input VCF or CSV files on to the 'Upload Files' page by simply dragging and dropping a file from their file explorer into the upload field. Variants are then parsed through a module to 
+standardise thier format and saved as a text file. The variants are then validated with the module to query VariantValidator to return the associated HGVS nomenclature, which is stored in the project
```
typo for their

ayana-manoj commented on README.md:
```diff
@@ -1,2 +1,70 @@
+HGVS genomic identifier for a query variant and then using it to query ClinVar. This ensures that a standaradised and validated term for the query variant is used to search ClinVar.
+[HGNC](https://www.genenames.org/about/) gene ID and gene symbol are also extracted from the VariantValidator search results to include in the annotation.  
+
+ClinVar Search is accessd by [installing locally](https://github.com/ayana-manoj/ClinVar_Search/tree/develop/docs).
```
typo for accessed

ayana-manoj reviewed (commented):
This README provides a clear structured overview of ClinVar Search. It has a logical flow and explains how the app works. There are some minor typos but other than that this is good. Well done :)

KamVya2926 committed:
Corrected typos and added codecov ticker to README

NuurAH committed:
Update README.md

editing some aspects of the readme

NuurAH committed:
Update README.md

installation guide for the docker

NuurAH reviewed (changes requested):
Added my changes to the readme.
I think we need to look at the license section, but other than that, I think it looks good!

NuurAH committed:
Update README.md

using $ affects formatting so I tried to add some escape characters to prevent unintended formatting changes

KamVya2926 committed:
Update README.md

Minor corrections on typos and structure

ayana-manoj committed:
Documentation Sprint #28

ayana-manoj committed:
Update README with Codecov integration

ayana-manoj committed:
Merge branch 'documentation_sprint' of https://github.com/ayana-manoj/ClinVar_Search into documentation_sprint

KamVya2926 committed:
Updated README.md with link to GitHub repo, overview workflow, and links to installation and user guides

KamVya2926 committed:
Add workflow image

NuurAH committed:
Update INSTALLATION.md

NuurAH committed:
Update README.md

NuurAH committed:
Update README.md

NuurAH committed:
Update README.md

fixed markdown formatting issue

NuurAH committed:
Update README.md

ayana-manoj committed:
Update INSTALLATION.md

NuurAH reviewed (changes requested):
Just make sure the installation and docker guide are the changes from Sprint3/develop as those have been recently changed
